11-node-frameworks/classworks/Alexey_Goremykin/server.js

```diff
@@ -0,0 +1,79 @@
+const serve = require('koa-static');
+const session = require('koa-generic-session');
+const bodyParser = require('koa-bodyparser');
+const fs = require('fs');
+const Joi = require('joi')
+
+const Koa = require('koa');
+const Router = require('koa-router');
+const app = new Koa();
+const router = new Router();
+
+let db = {};
+
+app.keys = ['keys', 'keykeys'];
+
+
+app.use(serve('static'));
+app.use(session());
+app.use(bodyParser());
+
+app.use(async (ctx, next) => {
+  ctx.body = 'Hello World';
+  next();
+});
+
+app.use((ctx, next) => {
+  ctx.body = 'My function'
+  next();
+});
+
+
+app.use(async (ctx, next) => {
+  if (ctx.path === '/get'){
+    get(ctx);
+  } else if (ctx.path === '/regenerate') {
+    regenerate(ctx);
+  }
+  next()
+});
+
+const get = (ctx) => {
+  let session = ctx.session;
+  session.count = session.count || 0;
+  session.count++;
```

Comment:
Viewers counter should work by default for every new endpoint in your code without adding new code. TIP: use pure middleware, pay attention to how body-parser works